movement/watch_faces/clock/simple_clock_face.c

```diff
@@ -161,11 +184,37 @@ void simple_clock_face_resign(movement_settings_t *settings, void *context) {
 }
 
 bool simple_clock_face_wants_background_task(movement_settings_t *settings, void *context) {
-    (void) settings;
     simple_clock_state_t *state = (simple_clock_state_t *)context;
     if (!state->signal_enabled) return false;
 
     watch_date_time date_time = watch_rtc_get_date_time();
 
-    return date_time.unit.minute == 0;
+    if (date_time.unit.minute == 0)
+        return true;
+
+    switch (settings->bit.signal_freq) {
+    case SIGNAL_1MIN:
+            return date_time.unit.second == 0;
```

Comment:
No need to check for `date_time.unit.second == 0` here and in the other cases. The `..._wants_background_task`-handlers are only called once a minute. Otherwise line the former line 170 would not have worked. I suggest you remove these checks, to keep things simple and streamlined.